mmv1/templates/terraform/examples/workbench_instance_full.tf.erb

```diff
@@ -0,0 +1,92 @@
+resource "google_workbench_instance" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['instance_name'] %>"
+  location = "us-central1-a"
+
+  gce_setup {
+    machine_type = "n1-standard-4"
+
+    accelerator_configs{
+      type         = "NVIDIA_TESLA_T4"
+      core_count   = 1
+    }
+
+    vm_image {
+      project      = "deeplearning-platform-release"
+      family       = "tf-latest-cpu"
+    }
+
+    service_accounts {
+      email = "<%= ctx[:test_env_vars]["service_account"] %>"
+    }
+
+    boot_disk {
+      disk_size_gb  = 110
+      disk_type = "PD_SSD"
+      disk_encryption = "GMEK"
+      kms_key = data.google_kms_crypto_key.crypto-key.id
+    }
+
+    data_disks {
+      disk_size_gb  = 110
+      disk_type = "PD_SSD"
+      disk_encryption = "GMEK"
+      kms_key = data.google_kms_crypto_key.crypto-key.id
+    }
+
+    shielded_instance_config {
+      enable_secure_boot = true
+      enable_vtpm = true
+      enable_integrity_monitoring = true
+    }
+
+    network_interfaces {
+      network = data.google_compute_network.my_network.id
+      subnet = data.google_compute_subnetwork.my_subnetwork.id
+      nic_type = "GVNIC"
+    }
+
+    disable_public_ip = true
+
+    tags = ["abc","def"]
+
+    metadata = {
+      terraform = "true"
+    }
+
+    enable_ip_forwarding = true
+
+    gpu_driver_config {
+      enable_gpu_driver = true
+      custom_gpu_driver_path = "test_path"
+    }
+  }
+
+  instance_owners  = [ "<%= ctx[:test_env_vars]["service_account"] %>"]
+
+  disable_proxy_access = true
+
+  labels = {
+    k = "val"
+  }
+
+}
+
+data "google_compute_network" "my_network" {
+  name = "test-default"
```

Comment:
Could you please update the `name` fields of the `google_compute_network` and `google_compute_subnetwork` resources so they use `vars` set in the YAML? I.e. something like `"<%= ctx[:vars]['instance_name'] %>"` here in the example file.

Reasoning: There's some templating magic that takes place on values supplied via that route which means if you set the `var` value to `"test-default"` the generated acceptance tests will set the resource name to `"tf-test-test-default-<random string>"`. That naming style is needed to ensure the name is unique and can be identified by automated cleanup in our test GCP project.